knudev-security/src/test/integration/java/ua/knu/knudev/knudevsecurity/service/AuthenticationServiceTest.java

```diff
@@ -1,12 +1,19 @@
-package ua.knu.knudev.knudevsecurity.service;
-
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.Test;
-
-class AuthenticationServiceTest {
-
-    @Test
-    void authenticate() {
-        Assertions.assertTrue(true);
-    }
-}
\ No newline at end of file
+//package ua.knu.knudev.knudevsecurity.service;
```

Comment:
delete

Reply:
i will very soon complete this test in new PR